srvutil/metrics.go

```diff
@@ -50,30 +51,22 @@ func RequestMetricsMiddleware(next http.Handler) http.Handler {
 		ctx = statsd.WatchingTagLoggable(ctx, recorder)
 		r = r.WithContext(ctx)
 
-		headers := &strings.Builder{}
-		// Ignore headers that may contains sensitive information.
-		r.Header.WriteSubset(headers, map[string]bool{
-			"Authorization": true,
-			"Cookie":        true,
-		})
+		reqHeaders := redact.Headers(r.Header)
 
 		log(ctx, nil).
 			WithField("method", r.Method).
-			WithField("headers", headers).
+			WithField("headers", reqHeaders).
 			Info("http request")
 
 		metrics.HTTPRequest.Time(ctx, func() error {
 			next.ServeHTTP(recorder, r)
 			return nil
 		})
 
-		headers = &strings.Builder{}
-		w.Header().WriteSubset(headers, map[string]bool{
-			"Set-Cookie": true,
-		})
+		resHeaders := redact.Headers(recorder.Header())
```

Comment:
This is using `recorder.Header()` where previously `w.Header()` was used. Why this change?

Reply:
No reason, will change back.